### ISSUE TITLE
Docker Docs

### ISSUE THREAD

**Issue by CJ-Wright** [regro/rever#99]
It might be nice to have some docs on how Docker needs to be set up to work with rever.

**Comment by scopatz** [comment]
Thanks for opening this @CJ-Wright. In what way do you mean? Like how to set env vars in the rever.xsh or how to install docker?

**Comment by CJ-Wright** [comment]
Is there a particular configuration needed for docker beyond just `<package-manager> install docker` and the env vars? (Eg docker needs to be set up to run as user)

**Comment by scopatz** [comment]
Yes, docker needs to be set up to be run as a user.  This is a pretty common thing, as far as I understand it.

**Comment by scopatz** [comment]
We can post that to the docs somewhere (I thought it was...) and then post a link to how to do that.
